### ISSUE TITLE
[hevc_nvenc @ 0000017407d70740] B frames as references are not supported

### ISSUE THREAD

**Issue by JCDinPGH** [GyanD/codexffmpeg#78]
Somewhere between the 10/13/2022 and 11/21/2022 builds, all encodes using similar as below fail with

command: ffmpeg -n -hwaccel cuda -threads 1 -i 1.mkv -c:s copy -c:a copy -map 0:a -map 0:s? -map 0:v -c:v hevc_nvenc -preset slow -qmax 20 "hevc-1.mkv"

[hevc_nvenc @ 0000017407d70740] B frames as references are not supported
[hevc_nvenc @ 0000017407d70740] Provided device doesn't support required NVENC features
Error initializing output stream 0:2 -- Error while opening encoder for output stream #0:2 - maybe incorrect parameters such as bit_rate, rate, width or height
Conversion failed!

I have tried this with at least 10 different files and they all fail with the same errors. After going back to the 10/13/2022 build, the conversions work fine. 

FWIW, I also tried the 11/27/2022 BtbN build ffmpeg-master-latest-win64-gpl.zip and it also fails on all 10 files I tried. I am working on narrowing it down to exactly where it started failing so give me some time. 

I reencode everything h.264 to x.265 with high quality settings and noticed this when replacing my existing ffmpeg binary with the 11/21/2022 build. Because this is a lot of files and the whole process is automated via a script in sonarr, I didn't notice it right away. 

I tested all of the full builds of
ffmpeg-2022-10-13-git-9e8a327e68-full_build
ffmpeg-2022-10-17-git-3bd0bf76fb-full_build
ffmpeg-2022-10-20-git-eb9153b4a7-full_build
ffmpeg-2022-10-24-git-d79c240196-full_build
ffmpeg-2022-10-27-git-00b03331a0-full_build
ffmpeg-2022-10-30-git-ed5a438f05-full_build
ffmpeg-2022-11-03-git-5ccd4d3060-full_build
ffmpeg-2022-11-23-git-c8e9cc8d20-full_build
and the first time I get the error listed above is on the 11/23/2022 build. All of the others work correctly on the same sample file I have been using as a test MKV.
I apologize as I know this isn't super scientific but I am trying to be as thorough as possible. I tried these on two different machines with two different GPUs and both have the v526_47 NVIDIA drivers.

**Comment by JCDinPGH** [comment]
I tested all of the full builds of
ffmpeg-2022-10-13-git-9e8a327e68-full_build
ffmpeg-2022-10-17-git-3bd0bf76fb-full_build
ffmpeg-2022-10-20-git-eb9153b4a7-full_build
ffmpeg-2022-10-24-git-d79c240196-full_build
ffmpeg-2022-10-27-git-00b03331a0-full_build
ffmpeg-2022-10-30-git-ed5a438f05-full_build
ffmpeg-2022-11-03-git-5ccd4d3060-full_build
ffmpeg-2022-11-23-git-c8e9cc8d20-full_build
and the first time I get the error listed above is on the 11/23/2022 build. All of the others work correctly on the same sample file I have been using as a test MKV.
I apologize as I know this isn't super scientific but I am trying to be as thorough as possible. I tried these on two different machines with two different GPUs and both have the v526_47 NVIDIA drivers.
